kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h

```diff
@@ -259,6 +259,7 @@ class ResidualBasedNewtonRaphsonStrategy
      
     virtual ~ResidualBasedNewtonRaphsonStrategy()
     {
+        Clear();
```

Comment:
What is being cleaned here? If it is just mpA, mpDx, mpb, I don't know if this is necessary, since those are smart pointer and will self-destruct anyway if no-one references them.